### PR TITLE
Update Money Manager EX (mmex) to latest stable version

### PR DESCRIPTION
**Pull request by alexweber**

### CODE REVIEW

File: Casks/mmex.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask :v1 => 'mmex' do
-  version '1.1.0'
-  sha256 '37042d0255e4bf89aecab1373e8c27597ad654f9164a1ee1d2773f021c79a925'
+  version '1.2.2'
+  sha256 'c9da531544af256490ec5d1a62080535c1440b10f80f3600c250c7cbd0afd269'
 
   # sourceforge.net is the official download host per the vendor homepage
-  url "http://downloads.sourceforge.net/sourceforge/moneymanagerex/Version%20#{version}/mmex_#{version}_mac.dmg"
+  url "http://downloads.sourceforge.net/project/moneymanagerex/moneymanagerex/Version_{version}/mmex_{version}_mac10.10.dmg"
```
Comment (adityadalal924) [inline Casks/mmex.rb]: should be `#{version}`

**Comment by alexweber** [comment]
@adityadalal924 oops, good catch, thanks! :)

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by alexweber** [comment]
@vitorgalvao You bet!  Coming soon....

**Comment by vitorgalvao** [comment]
Please @mention me after you do, so I get an email to check back.

**Comment by alexweber** [comment]
@vitorgalvao There ya go, thx!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as e40a62e7aebc1d45d70a348d2fdeba597fdbd1ea to keep commit history cleaner. Your contribution is still credited to you.

**Comment by alexweber** [comment]
:+1:

**Pull request closed without merging**